script/packagecloud-prune.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+PACKAGECLOUD_USER="${PACKAGECLOUD_USER:-github}"
+PACKAGECLOUD_REPO="${PACKAGECLOUD_REPO:-orchestrator}"
```

Comment:
Wrong repository.

Reply:
I'd suggest not using `packagecloud` at this time.

Reply:
i use packagecloud for my own repo for testing orchestrator-agent on my staging stang
and use packagecloud for install orchestrator

would i rename PACKAGECLOUD_REPO to default orchestrator-agent?